native/packages/react-native-bpk-component-switch/src/__snapshots__/BpkSwitch-test.android.js.snap

```diff
@@ -64,11 +61,6 @@ exports[`Android BpkSwitch should support theming 1`] = `
   onStartShouldSetResponder={[Function]}
   onTintColor="rgb(230, 228, 235)"
   style={undefined}
-  theme={
```

Comment:
It looks like the theme functionality on the switch has been regressed...

Reply:
Nope it just used to be spread on the Switch element as part of `rest` before, but now it isn't.

Reply:
Ok then in that case we must be missing a test for theming? For instance I can't see any application of a "theme" in these snapshots: https://github.com/Skyscanner/backpack/blob/76b1c7b0a028f312c46b8d15ad57c97cae6a094d/native/packages/react-native-bpk-component-switch/src/__snapshots__/BpkSwitch-test.android.js.snap

Reply:
The `theme` prop never ends up in the snapshot for any of our themeable components instead other props/styles are derived and passed down. In this case this is captured by the different tint colros